site/content/entry/yuzu-progress-report-apr-2020/index.md

```diff
@@ -0,0 +1,208 @@
++++
+date = "2020-05-28T12:00:00-03:00"
+title = "Progress Report April 2020"
+author = "GoldenX86"
+forum = 243216
++++ 
+
+Hello awesome yuz-ers! We hope you are doing well. 
+We have lots of material to cover in this April progress report: some new additions, bug and regression fixes, and the groundwork for future improvements that affect both Mainline and Early Access. But first, let's discuss a few really big changes.
+<!--more-->
+
+## The Big Changes
+
+This month brought us huge improvements to compatibility, fidelity and resource management. 
+[bunnei](https://github.com/bunnei), our serendipitous master, fixed a softlock in `FINAL FANTASY VIII Remastered`. 
+What he didn’t know at the time is that this small fix also made `Super Smash Bros. Ultimate` playable. Thanks to this, now you can enjoy local multiplayer battles with your friends, either locally or even via Parsec!
+
+For quite a while, Nvidia Turing users (GTX 16 series and RTX 20 series GPUs) had to deal with several rendering bugs, like black and white terrain in `Fire Emblem: Three Houses`, or black areas in `Super Monkey Ball: Banana Blitz HD`. 
+[Rodrigo](https://github.com/ReinUsesLisp) fixed most of them by improving the accuracy of the shader decoder. This also fixes some bugs on older Nvidia GPUs too.
+
+{{< single-title-imgs
+    "Monochrome no more (Fire Emblem: Three Houses)"
+    "./29.png"
+    "./30.png"
+  >}}
+
+{{< single-title-imgs
+    "No greasy black peels (Super Monkey Ball: Banana Blitz HD)"
+    "./27.png"
+    "./28.png"
+  >}}
+  
+Speaking of shader decoding accuracy, [Rodrigo](https://github.com/ReinUsesLisp) improved the precision of the `HADD2` and `HMUL2` instructions, fixing some long standing bugs like the fog in `Fire Emblem: Three Houses` and the excessive brigthness in most maps in `Super Smash Bros. Ultimate`.
+
+{{< single-title-imgs
+    "You can now take off your sunglasses (Super Smash Bros. Ultimate)"
+    "./33.png"
+    "./34.png"
+  >}}
+  
+{{< single-title-imgs
+    "From steampunk, to the proper medieval fantasy setting (Fire Emblem: Three Houses)"
+    "./16.png"
+    "./15.png"
+  >}}
+
+We talked about this in its own [article](https://www.patreon.com/posts/35856792), but it deserves a mention here:
+[bunnei](https://github.com/bunnei) rewrote the Virtual Memory Manager (known internally as `Project Epimetheus`), which drastically reduced RAM use by games, and helped Multicore support (also known as`Project Prometheus`) become a reality.
```

Comment:
Some users do care about homebrew, so I'd add this sentence back in at the end of this paragraph:

`This rewrite also resolved a crashing issue in some homebrew games/apps.`